doc/classes/NavigationObstacle2D.xml

```diff
@@ -66,10 +66,10 @@
 			Sets the avoidance radius for the obstacle.
 		</member>
 		<member name="velocity" type="Vector2" setter="set_velocity" getter="get_velocity" default="Vector2(0, 0)">
-			Sets the wanted velocity for the obstacle so other agent's can better predict the obstacle if it is moved with a velocity regularly (every frame) instead of warped to a new position. Does only affect avoidance for the obstacles [member radius]. Does nothing for the obstacles static vertices.
+			Sets the wanted velocity for the obstacle so other agents can better predict the obstacle if it is moved with a velocity regularly (every frame) instead of warped to a new position. Does only affect avoidance for the obstacles [member radius]. Does nothing for the obstacle's static vertices.
 		</member>
 		<member name="vertices" type="PackedVector2Array" setter="set_vertices" getter="get_vertices" default="PackedVector2Array()">
-			The outline vertices of the obstacle. If the vertices are winded in clockwise order agents will be pushed in by the obstacle, else they will be pushed out. Outlines can not be crossed or overlap. Should the vertices using obstacle be warped to a new position agent's can not predict this movement and may get trapped inside the obstacle.
+			The outline vertices of the obstacle. If the vertices are winded in clockwise order agents will be pushed in by the obstacle, otherwise they will be pushed out. Outlines can not be crossed or overlap. Should the obstacle be warped to a new position agents can not predict this movement and may get trapped inside the obstacle.
```

Comment:
Same for 3D I think.
```suggestion
			The outline vertices of the obstacle. If the vertices are winded in clockwise order, agents will be pushed in by the obstacle, otherwise they will be pushed out. Outlines cannot be crossed or overlap. If the obstacle is warped to a new position, agents cannot predict this movement and may get trapped inside the obstacle.
```